src/dump.rs

```diff
@@ -87,7 +87,7 @@ impl RustDocCommand {
             }
         }
 
-        let json_path = target_dir.join(format!("doc/{}.json", crate_name));
+        let json_path = target_dir.join(format!("doc/{}.json", crate_name.replace('-', "_")));
```

Comment:
This looks good. In case there's a problem with the crate name -> JSON file lookup expectations themselves, consider including the crate name in the `anyhow::bail!()` call together with the `json_path`.